### PR TITLE
Updated ReadWrite Lock to enforce best practice

### PR DESCRIPTION
**Pull request by vikas171999**
- Corrected Spelling mistake
- Removed unneeded self
- made concurrentQueue private as it is not used outside that class. And as apple suggests if we are not using anything outside the class we should always make it private. We should upgrade the access level as and when the variable needed outside

### CODE REVIEW

File: Sources/ReadWriteLock.swift [hunk 1]
```diff
@@ -8,19 +8,19 @@
...
+    private let concurrentQueue: DispatchQueue
 
     init(label: String) {
-        self.concurentQueue = DispatchQueue(label: label, qos: .utility, attributes: .concurrent)
+        concurrentQueue = DispatchQueue(label: label, qos: .utility, attributes: .concurrent)
     }
 
     func read(closure: () -> Void) {
-        self.concurentQueue.sync {
+        concurrenttQueue.sync {
```
Comment (jaredmixpanel) [inline Sources/ReadWriteLock.swift:18]:
```suggestion
        concurrentQueue.sync {
```

**Review by jaredmixpanel — CHANGES_REQUESTED** [review]
small typo and please also pull in the latest changes from master, to get updated CI config.

File: Sources/ReadWriteLock.swift (hunk 1 above)
Comment (vikas171999) [reply Sources/ReadWriteLock.swift]: Done

**Comment by jaredmixpanel** [comment]
@vikas171999 CI is still failing. Please merge the latest upstream master to get the updates to our CI config.

**Comment by jaredmixpanel** [comment]
@vikas171999 can you please pull in the latest changes from master? they include some CI config changes that will fix your CI failure.

**Pull request closed without merging**